### DISCUSSION TITLE
Are XLabs Flux models safe?

### DISCUSSION THREAD

**Discussion by billnye2** [comfyanonymous/ComfyUI#4261] [General]
[There's been some concern](https://www.reddit.com/r/StableDiffusion/comments/1em9u6x/first_flux_controlnet_canny_was_just_released_by/) whether the models by XLabs for Flux (controlnet and realism lora) are safe to use. I saw you (comfy) added a version of the lora converted for comfyui on hf. In your opinion, are the models safe to use? Thank you

**Comment by comfyanonymous** [comment]
safetensors is a safe format so you don't have to worry.

**Reply by billnye2** [reply]
Right yeah, someone mentioned in that thread that there were previous safetensors files for some unrelated thing that could still be loaded as pt file. [Here's a link to what they said](https://www.reddit.com/r/StableDiffusion/comments/1em9u6x/comment/lgz6byx/?utm_source=share&utm_medium=web3x&utm_name=web3xcss&utm_term=1&utm_content=share_button), it's the case where the code that loads the safetensors file fallsback to loading as pt if it detects the content type of the file. Hoping comfyui doesn't do this and that it would only load a .safetensors file using the hf lib and otherwise fail correctly
